librarian_packages/hera_librarian/tests/test_base_store.py

```diff
@@ -0,0 +1,183 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2019 the HERA Collaboration
+# Licensed under the 2-clause BSD License
+
+"""Test code in hera_librarian/store.py
+
+"""
+
+from __future__ import print_function, division, absolute_import
+import pytest
+import os
+import json
+from hera_librarian import base_store, RPCError
+
+from . import ALL_FILES, filetypes, obsids, md5sums, pathsizes
+
+
+@pytest.fixture
+def local_store():
+    return base_store.BaseStore("local_store", "/tmp", "localhost")
+
+
+def test_path(local_store):
+    # test that store path is prepended
+    assert local_store._path("my_dir") == "/tmp/my_dir"
+
+    # test passing in an absolute path
+    with pytest.raises(ValueError):
+        local_store._path("/tmp/my_dir")
+
+    return
```

Comment:
A bare `return` statement at the end of a function isn't necessary. (Sometimes I do put statements like this to give a reference point for the indentation, though.)

Reply:
I know, I just like to be explicit about things :)